packages/dashmate/src/commands/setup.js

```diff
@@ -87,66 +107,61 @@ class SetupCommand extends BaseCommand {
       },
     ],
     {
+      concurrent: false,
       renderer: isVerbose ? 'verbose' : 'default',
       rendererOptions: {
         showTimer: isVerbose,
         clearOutput: false,
         collapse: false,
         showSubtasks: true,
+        removeEmptyLines: false,
       },
     });
 
+    if (!isVerbose) { // TODO: We need to print it only with default renderer
+      const { begoo } = await import('begoo');
+
+      const welcomeText = begoo(
+        chalk`Hello! I'm your {bold.cyanBright Dash} mate!
+
+I will assist you with setting up a Dash node on mainnet or testnet. I can also help you set up a development network on your local system.`,
+        { maxLength: 45 },
+      );
+
+      // eslint-disable-next-line no-console
+      console.log(welcomeText);
+    }
+
     try {
       await tasks.run({
         preset,
-        nodeType,
         nodeCount,
         debugLogs,
         minerInterval,
-        externalIp,
-        operatorBlsPrivateKey,
-        fundingPrivateKeyString,
         isVerbose,
-        zeroSslApiKey,
-        certificateProvider,
-        sslCertificateFilePath,
-        sslCertificatePrivateKeyFilePath,
       });
     } catch (e) {
       throw new MuteOneLineError(e);
     }
   }
 }
 
-SetupCommand.description = `Set up node config
+SetupCommand.description = `Setup a new Dash node
```

Comment:
I noticed some places we are using "setup" and other places "set up". I think either is fine, but we should try to be consistent.

Reply:
"setup" is a noun, as in "run the setup command, or "the meeting with the undercover cops was a setup". "set up" is a verb, as in "set up the node" or "set someone up on a date". Does this hold in US English too?